snc-library.sh

```diff
@@ -189,7 +189,11 @@ function renew_certificates() {
     start_vm ${vm_prefix}
 
     # After cluster starts kube-apiserver-client-kubelet signer need to be approved
-    timeout 500s bash -c -- "until ${OC} get csr | grep Pending; do echo 'Waiting for first CSR request.'; sleep 2; done"
```

Comment:
Maybe keep this line around for catching ths CSRs other then `kube-apiserver-client-kubelet` and `kubelet-serving`? now we are filtering on these two names only and there is also the `kube-apiserver-client` CSR, not sure if there can be more!

Reply:
@anjannath as of now I think we are only care about `kube-apiserver-client-kubelet` and `kubelet-serving` because other csr automatic approved by machine approver. Actually we only care about `kube-apiserver-client-kubelet` but as of now this is just make sure if not approved then we do it.

Reply:
Okay, thanks for explaining!